README.md

```diff
@@ -1,58 +1,48 @@
+- Develop environment
+  - Visual Studio 2015
+- Run-time environment
+  - .NET Framework 4.6
 
-## 概要(summary)
-以下のファイルを参照してください。
-(Please refer to the following files.)
+If you want to use the latest development environment (e.g. Visual Studio 2015), please refer [Templates base](https://github.com/OpenTouryoProject/OpenTouryoTemplatesV2).
 
+## Summary
+Please refer to the following files.
  - [Introduction.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pdf)
  - [Introduction.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Introduction.pptx)
  - [Introduction_EN.pdf](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pdf)
  - [Introduction_EN.pptx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/en/Introduction_EN.pptx)
  - [Functional_list.xlsx](https://github.com/OpenTouryoProject/OpenTouryoDocuments/blob/master/documents/0_Introduction/ja-JP/Functional_list.xlsx)
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
 
-##詳細(details)
-以下のディレクトリを参照してください。
-(Please refer to the following directories.)
+## Details
+Please refer to the following directories.
 
  - [documents](https://github.com/OpenTouryoProject/OpenTouryoDocuments/tree/master/documents)
 
-ドキュメント類は[OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments)リポジトリに格納されています。
-(documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.)
+Documents are located in the [OpenTouryoProject/OpenTouryoDocuments](https://github.com/OpenTouryoProject/OpenTouryoDocuments) repository.
 
-##内容物(contents)
+## Contents
 
-###ディレクトリ(directory)
+### Directory
 
 #### [/license/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/license)
-ライセンス(license)
+License
 
 #### [/root/](https://github.com/OpenTouryoProject/OpenTouryo/tree/master/root)
-プログラム
+Program
 
-##テンプレート・ベース(Templates base)
-テンプレート・ベースに同梱されるサンプルはOpen棟梁の評価に利用できます。
-また、テンプレート・ベースをチュートリアルの内容に従いカスタマイズすることで
-当該Visual Studioバージョンの案件向けプロジェクト・テンプレートを作成できます。 
-
-(Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
+## Templates base
+Samples that are shipped with the templates base can be used to evaluate the OpenTouryo. 
```

Comment:
@daisukenishino 

Please revise as follows:

*Open Touryo Template Base* is the *foundation* of the development infrastructure (project template) of the programs using Open Touryo.
The samples included in *Open Touryo Template Base* can be used to evaluate Open Touryo.